build/package/Dockerfile.finish

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV GO_VERSION 1.16.3
+
+RUN yum install -y git
+
+RUN cd /tmp && \
+    curl -LO https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go$GO_VERSION.linux-amd64.tar.gz && \
+    rm -f *.tar.gz && \
+    cd - && \
+    mkdir /go
+
+ENV PATH $PATH:/usr/local/go/bin
+ENV GOBIN /usr/local/bin
+
+RUN chmod g+w /go
+
+# Add Go binaries
+RUN mkdir -p /etc/go
+COPY go.* /etc/go/
+COPY cmd /etc/go/cmd
+COPY internal /etc/go/internal
+COPY pkg /etc/go/pkg
+RUN cd /etc/go/cmd/finish && CGO_ENABLED=0 go build -o /usr/local/bin/ods-finish
+
+WORKDIR /go
```

Comment:
Could you configure your editor to include EOF newlines? That avoids unnecessary diff when the file is edited again with EOF newlines configured ;)